CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+## 3.4.1 
+- Added support for [Collaboration Whitelist](https://github.com/box/box-windows-sdk-v2/blob/master/Box.V2/Managers/BoxCollaborationWhitelistManager.cs) endpoint
+- Added [Event Types Enum](https://github.com/box/box-windows-sdk-v2/blob/master/Box.V2/Config/Constants.cs#L287)
+- Fixed deserialization issue with BoxRepresentationStatus(#429)
```

Comment:
It doesn't look like GitHub picked up on the Issue reference here; maybe try adding a space before the opening paren on all these?

Reply:
let me know if it works now @mattwiller thanks!